README.md

```diff
@@ -57,7 +57,23 @@ self.addEventListener('periodicsync', event => {
 
 # Security and Privacy
 
-TODO
+One-shot Background Sync allows the web page’s logic to live a little longer (service worker execution time) after the page has been closed by the user. Periodic Background Sync extends it to potentially infinite time thereafter. Here are some privacy concerns:
```

Comment:
Please settle on either using line breaks at a particular line length limit, or not having a line length limit at all, and be consistent throughout the document.

Reply:
Sorry, my editor had a word wrap, which I think was being applied when viewing the doc only. I've added manual line breaks at or below 100 columns now.